draft-ietf-ice-pac.xml

```diff
@@ -238,6 +238,32 @@
         pairs prior to the timer elapsing.
       </t>
     </section>
+    
+    <section title="Update to Trickle ICE">
```

Comment:
s/Trickle ICE/RFC XXX

RFC XXX will then be replaced with the RFC number assigned to trickle.